pandas/tests/scalar/test_timestamp.py

```diff
@@ -579,47 +587,42 @@ def check(value, equal):
     def test_nat_fields(self):
         # GH 10050
         ts = Timestamp('NaT')
-        self.assertTrue(np.isnan(ts.year))
-        self.assertTrue(np.isnan(ts.month))
-        self.assertTrue(np.isnan(ts.day))
-        self.assertTrue(np.isnan(ts.hour))
-        self.assertTrue(np.isnan(ts.minute))
-        self.assertTrue(np.isnan(ts.second))
-        self.assertTrue(np.isnan(ts.microsecond))
-        self.assertTrue(np.isnan(ts.nanosecond))
-        self.assertTrue(np.isnan(ts.dayofweek))
-        self.assertTrue(np.isnan(ts.quarter))
-        self.assertTrue(np.isnan(ts.dayofyear))
-        self.assertTrue(np.isnan(ts.week))
-        self.assertTrue(np.isnan(ts.daysinmonth))
-        self.assertTrue(np.isnan(ts.days_in_month))
+
+        for field in self._field_accessors:
+
+            result = getattr(NaT, field)
+            assert np.isnan(result)
+
+            result = getattr(ts, field)
+            assert np.isnan(result)
+
+        for field in self._bool_accessors:
+
+            result = getattr(NaT, field)
+            assert not result
```

Comment:
`assert result is False` 
In the bare assert style for pytest, I find that a bit clearer if you are explicitly testing for False, and in this case, more importantly, otherwise 0 would pass the test (eg if for some reason the result if converted to float which is what happens in the current code)